core_debug.tex

```diff
@@ -37,6 +37,25 @@ \section{Load-Reserved/Store-Conditional Instructions}
 may be no forward progress if Debug Mode is entered between {\tt lr} and {\tt
 sc} pairs.
 
+\section{Single Step}
+
+A debugger can cause a halted hart to execute a single instruction and then
+re-enter Debug Mode by setting \Fstep before setting \Fresumereq.
+
+If executing or fetching that instruction causes an exception, Debug Mode is
+re-entered immediately after the PC is changed to the exception handler and the
+appropriate {\tt badaddr} and {\tt cause} registers are updated.
```

Comment:
I just remembered that "*badaddr" is now "*tval" in the latest priv spec, so we should use that name instead